cogboard-local-compose.yml

```diff
@@ -18,6 +18,8 @@ services:
 
   backend:
     image: "cogboard/cogboard-app:${COGBOARD_VERSION}"
+    environment:
+      - COGBOARD_VERSION=${COGBOARD_VERSION}
```

Comment:
The Cogboard version is now provided by docker as a system environment variable.